molgenis-data-jpa/src/test/java/org/molgenis/data/jpa/JpaImportTest.java

```diff
@@ -158,7 +158,7 @@ public void testMrefWrong() throws IOException
 		{
 			assertEquals(e.getViolations().size(), 1);
 			assertEquals(e.getViolations().iterator().next().getViolatedAttribute().getName(), "children");
-			assertEquals(e.getViolations().iterator().next().getRownr(), 3);
+			assertEquals(e.getViolations().iterator().next().getRownr(), Long.valueOf(3));
```

Comment:
But the JPA unit tests sill check for the same row numbers. Were their row numbers incorrect as well?